packages/node_modules/@webex/internal-plugin-user/test/integration/spec/user.js

```diff
@@ -141,7 +141,8 @@ runs.forEach((run) => {
         assert.isUndefined(unauthWebex.credentials.supertoken);
         const email = `collabctg+webex-js-sdk-${uuid.v4()}@gmail.com`;
 
-        return unauthWebex.internal.user.verify({email})
+        return unauthWebex.internal.services.collectPreauthCatalog({email})
+          .then(() => unauthWebex.internal.user.verify({email}))
```

Comment:
Would it make sense to have `verify` wait for the `collectPreauthCatalog` function so you don't have to do it here in the test?

Reply:
I agree with this after reviewing the details of the method. I have updated the src files to reflect this.